src/registrar/views/domain.py

```diff
@@ -335,6 +336,8 @@ def _make_invitation(self, email_address):
             )
         else:
             # created a new invitation in the database, so send an email
+            dapplication = DomainApplication.objects.filter(approved_domain=self.object)
```

Comment:
We want to use DomainInformation or Domain here. Context: DA gets stale after it gets approved and a Domain + DI get created.

Reply:
I would also move the full_name = domain_information.first.creator here and set "full_name": full_name below. Slightly easier to follow I think.

Reply:
I will piggy back and agree with the above